lib/hmmlearn/hmm.py

```diff
@@ -1213,7 +1211,10 @@ def _do_mstep(self, stats):
             # section 3.2
             # https://vioshyvo.github.io/Bayesian_inference
             alphas, betas = self.lambdas_prior, self.lambdas_weight
-            n = stats['nsamples']
-            kappas = betas / (betas + n)
+            n = stats['post'].sum()
```

Comment:
Actually, that's just `stats["nobs"]`, I think?

Reply:
Ah, I see.  That's a bit confusingly named, and moreover, never used afaict; probably we should just get rid of it at some point.

Reply:
Agreed and quite, I originally tried to use it and was pretty confused